src/storage/src/storage_failpoints/test_iterator.rs

```diff
@@ -396,13 +395,14 @@ async fn test_failpoints_compactor_iterator_recreate() {
     .await
     .unwrap();
 
+    info.table_ids.push(table_id as u32);
```

Comment:
why not make the `put_sst` implementation populae the table_ids field correctly?